### ISSUE TITLE
Command line interface

### ISSUE THREAD

**Issue by Bamban** [wavedrom/wavedrom#165]
Hi ! First of all thanks a lot for this awesome tool.
I'm using it to generate waveforms for some documentation. I currently have some json wave files in a directory. In order to process them with latex, I need pdf. I can easily go from svg to pdf (which is great because everything is vector) with inkscape. The only thing I can't automate is the export of json to svg. I currently open the editor and click "export svg".

Hence my question : would it be possible to imagine a command line interface doing something like this ?
`./wavedrom --in wave.json --export-svg --out wave.svg`

Thanks for your time !

**Comment by drom** [comment]
Hi @Bamban . Thank you for the kind words. Here is CLI version of wavedrom https://github.com/wavedrom/cli Please try.

**Comment by 3DES** [comment]
CLI is really a great idea!

Any chance to get it implemented directly into the standalone waveformeditor, since the editor has all needed options already implemented and especially since PhantomJS project has been suspended?

Thanks a lot spending your time!

**Comment by drom** [comment]
Yes, It is possible since `chromium` has `headless` mode now: https://developers.google.com/web/updates/2017/04/headless-chrome

And if `NWJS` supports it I will implement CLI for it.
https://github.com/nwjs/nw.js/issues/769
